src/containers/org/EditMetadataBody.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from 'react';
+
+import styled from 'styled-components';
+import {
+  Input,
+  Label,
+  Typography,
+} from 'lattice-ui-kit';
+
+import { ModalBody } from '../../components';
+
+const FieldWrapper = styled.div`
+  margin-bottom: 16px;
+`;
```

Comment:
I created `StackGrid` to avoid this type of thing.

Reply:
yeah but then i'd have to group the labels and the inputs in another inner wrapper if i wanna use the stack grid. since it's just for a single group, i don't feel bad.

Reply:
ok fine.